_api-reference/cluster-api/shard-stores.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: default 
+title: Shard stores 
+parent: Cluster APIs 
+nav_order: 70
+---
+
+# Shard stores
+
+The `_shard_stores` API provides information about the shard copies for one or more indices. This API helps to diagnose issues with unallocated shards by providing reasons why shards are unassigned and their current states.
+
+<!-- spec_insert_start
+api: indices.shard_stores
+component: endpoints
+-->
+## Endpoints
+```json
+GET /_shard_stores
+GET /{index}/_shard_stores
+```
+<!-- spec_insert_end -->
+
+<!-- spec_insert_start
+api: indices.shard_stores
+component: path_parameters
+include_global: false
+include_deprecated: false
+pretty: false
+-->
+## Path parameters
+
+The following table lists the available path parameters. All path parameters are optional.
+
+| Parameter | Data type | Description |
+| :--- | :--- | :--- |
+| `index` | List or String | List of data streams, indexes, and aliases used to limit the request. |
+
+<!-- spec_insert_end -->
+
+<!-- spec_insert_start
+api: indices.shard_stores
+component: query_parameters
+include_global: false
+include_deprecated: false
+columns: Parameter, Data type, Description, Default
+pretty: true
+-->
+## Query parameters
+
+The following table lists the available query parameters. All query parameters are optional.
+
+| Parameter | Data type | Description | Default |
+| `allow_no_indices` | Boolean | If `false`, the request returns an error if any wildcard expression, index alias, or `_all` value targets only missing or closed indexes. This behavior applies even if the request targets other open indexes. | `false` |
+| `expand_wildcards` | List or String | Type of index that wildcard patterns can match. If the request can target data streams, this argument determines whether wildcard expressions match hidden data streams. <br> Valid values are: <br> - `all`: Match any index, including hidden ones. <br> - `closed`: Match closed, non-hidden indexes. <br> - `hidden`: Match hidden indexes. Must be combined with open, closed, or both. <br> - `none`: Wildcard expressions are not accepted. <br> - `open`: Match open, non-hidden indexes. | `open`  |
```

Comment:
"Must be combined with `open`, `closed`, or both" (code font)?

Reply:
Thats now pushed in this commit to PR in API-spec https://github.com/opensearch-project/opensearch-api-specification/pull/931/commits/88e556fcdea60dada4e0d98178330c74eb1754e2